snaps/index.mdx

```diff
@@ -78,6 +78,12 @@ Features include:
       href: "reference/permissions#endowmentnetwork-access",
       title: "Network access",
       description: <>Make API calls using <code>fetch()</code>.</>
+    },
+    {
+      href: "reference/rpc-api#snap_getlocale",
```

Comment:
Needs an icon

cc @eriknson

Reply:
He already identified the `home` icon from the DS to be used for the permission, so we just need to find that icon and use it here

Reply:
Just pushed an update with the correct icon!